root/layout/components/sidebar/ReleaseSidebar.js

```diff
@@ -61,7 +61,7 @@ const ReleaseSidebar = ({$c, release}: Props) => {
     ? new URL(releaseCoverUrl).host
     : null;
   const barcode = formatBarcode(release.barcode);
-  const typeName = releaseGroup.typeName;
+  const typeName = releaseGroup.l_type_name;
```

Comment:
This is a combined type name, which we don't translate. Elsewhere, we're just making the server serve an l_type_name - I think that's the most reasonable option here too.